manpages/lumina-xconfig.8

```diff
@@ -1 +1,37 @@
-Dummy file
+.Dd November 3, 2017
+.Dt LUMINA-XCONFIG 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-xconfig
+.Nd a graphical front-end to xrandr.
+It is used to manage monitor configurations.
```

Comment:
shorten and make active: "graphical front-end to xrandr, which manages monitor configurations."